src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -57,14 +57,25 @@ public enum LanguageVersion
         /// </list> 
         /// </remarks> 
         CSharp6 = 6,
+
         /// <summary>
         /// C# language version 7.
         /// </summary>
         CSharp7 = 7,
+
```

Comment:
It's a separate question, but do we want to make `CSharp7 = 70` instead, to make room for 7.1, etc?